mkosi

```diff
@@ -2567,6 +2567,8 @@ def parse_args() -> CommandLineArguments:
     group.add_argument('-f', "--force", action='count', dest='force_count', default=0, help='Remove existing image file before operation')
     group.add_argument('-b', "--bootable", type=parse_boolean, nargs='?', const=True,
                        help='Make image bootable on EFI (only gpt_ext4, gpt_btrfs, gpt_squashfs, gpt_xfs)')
+    group.add_argument("--boot-protocols", choices=("uefi", "bios"), default=[], nargs="+",
+                       help="Boot protocols to use on a bootable image", metavar="PROTOCOLS")
```

Comment:
Then error out on passing --bootable but having no boot-protocol?  I'd expect people to use uefi and only used they bios-only or hybrid version in awful scenarios like I have of supporting old hardware.

Reply:
I will update to use CommaDelimitedListAction, but I'm unsure about creating the dependencies here. For me it's nice to leave the boot protocols in the config file and decide if I'm creating a bootable/non-bootable image in the command line. If one option implies the other I think it will be tricky.

Reply:
i think we should default to uefi here, really. both because that's clearly the future, and because of compat